### PR TITLE
[bullet] Basic Features

### PR DESCRIPTION
**Pull request by Lobotuerk**
Signed-off-by: Tomas Lorente <jtlorente@ekumenlabs.com>
-Fixed position by adding the base model position into the equiation.
-Fixed gravity by adding an empty collisionshape into the bodies.
-Added a motion state to get the position out of bullet.
-Changed some autos into their proper variable.

### CODE REVIEW

File: bullet/src/SDFFeatures.cc [hunk 1]
```diff
@@ -60,17 +60,20 @@ Identity SDFFeatures::ConstructSdfLink(
       convertVec(ignition::math::eigen3::convert(diagonalMoments));
 
   // (TO-DO: do those calculation need to take into account in some way the base model?)
```
Comment (Blast545) [inline bullet/src/SDFFeatures.cc:62]: Can you remove this comment? I think you addressed this out here

**Review by Blast545 — APPROVED** [review]
LGTM, minor comment

File: bullet/src/SDFFeatures.cc (hunk 1 above)
Comment (Lobotuerk) [reply bullet/src/SDFFeatures.cc]: Done on a4c0c69

**Comment by Lobotuerk** [comment]
Merging after approval

**Pull request merged**